products/cpdb/python/projects_in_geographies.py

```diff
@@ -64,11 +64,3 @@ def export_table(geography_type: str, table_name: str) -> None:
             geography.geography_name,
         )
         export_table(geography.geography_type, geography.table_name)
-
-    # zip folder with all csv files
-    print(f"Zipping folder\n\t{OUTPUT_DIR}")
```

Comment:
does this affect the zipfile on BYTES?

Reply:
since this was creating a zip within the primary zip, it shouldn't impact metadata/distribution

we haven't been distributing these csvs so I don't think the file `projects_in_geographies.zip` (now left unzipped) was specified in metadata